docs/auth/atlassian/provider.md

```diff
@@ -46,18 +46,16 @@ auth:
       development:
```

Comment:
Would it make sense to include all the config here? There are some items that are missing, though optional, that would be good to see here. 

https://github.com/backstage/backstage/blob/master/plugins/auth-backend-module-atlassian-provider/config.d.ts

Reply:
@awanlin - added the optional parameters - thanks for the link that made it easy!